ultimatepython/classes/encapsulation.py

```diff
@@ -55,12 +64,12 @@ def __set_account_number(self, number):
         """
         self.__account_number = number
 
-    def delete_account(self):
+    def remove_account_details(self):
         """
         This method is used to delete an account.
```

Comment:
Fix docstring - we're not deleting account anymore

Reply:
Fixed.